### ISSUE TITLE
Feature: Add an option not to include date and time stamp and a separator 

### ISSUE THREAD

**Issue by magi44ken** [theohbrothers/ConvertOneNote2MarkDown#19]
Is it possible to add an option if they want to include the date, time stamp, and a separator after the title on the top of md note? For me I like to not include it and just the title is ok.

**Comment by nixsee** [comment]
hi, again I dont have time to do this. If you or someone else would like to submit a pull request, you can do so. It shouldn't be hard to follow the code and copy the format of the other options that are available to wrap the YAML section in an if statement. Or you could use a text editor to find/replace something like "*---" (which should select the whole header section that ends with "---" with nothing.

**Comment by nixsee** [comment]
@magi44ken Please continue over at https://github.com/theohbrothers/ConvertOneNote2MarkDown-v2

**Comment by leojonathanoh** [comment]
hey @magi44ken the feature is implemented in `v2.9.0`.

**Comment by nixsee** [comment]
@leojonathanoh you guys are killing it! Where were you a year ago to make all of this, and save me a few weeks of terrible hacking? Oh well, I did my part - I hope that the "foundation" I laid is at least somewhat useful still (though the real brains behind this was the original dev...)

**Comment by leojonathanoh** [comment]
@nixsee where were you when you first created this? You could have saved me a lot of headache if you had told me to move stuff off onenote so I dont have to go through the pain of now having to migrate so many notebooks. I appreciate your work very much :)
